content/app/development/configuration/email/_index.en.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Email
+description: How to create custom email notifications for apps.
+toc: true
+weight: 400
+---
+
+## Activate generation of email notifications in your application
+
+The email client is automatically added to the application. To use it, inject the "IEmailNotificationClient" interface. The interface defines a method used to order an email notification from the [Altinn Notifications](https://github.com/Altinn/altinn-notifications) API.
```

Comment:
Consider `IEmailNotificationClient` instead of "IEmailNotificationClient" since it references code